history-of-the-quakers/original/23-book-2-chapter-14.adoc

```diff
@@ -1,15 +1,21 @@
-== Chapter XIV. NEW ENGLAND.
-
-Discovery of North-America.--First Planters of New-England.
---John Robinson`'s Farewell-Admonition.--Emigrations
-multiply in the Reign of Charles I.--Reflection on the Spirit of these Colonists.--Roger
-Williams banished for his religious Opinions.--Schism at Boston.--Synod convened to
-decide upon it.--John Wheelwright summoned before the Court,
-disfranchised and banished.--John Coggeshal disfranchised.--Ann
-Hutchinson adopts Antinomian Opinions;
-summoned before the Court and condemned to Banishment;
-murdered by the Indians.--Some of the People professing Anabaptism are persecuted,--Three
-of them committed to Prison.--Law against Anabaptists.
+== Chapter XIV. New England.
+
+[.chapter-synopsis]
+* Discovery of North-America.--First Planters of New-England
```

Comment:
I think you missed one here, at `--First Planters<etc.>`

Reply:
That's so crazy because I feel like I get so meticulous with these! I check and recheck them!!